lnwallet/channel.go

```diff
@@ -631,6 +636,16 @@ type LightningChannel struct {
 	localUpdateLog  *updateLog
 	remoteUpdateLog *updateLog
 
+	// pendingFeeUpdate is set to the fee-per-kw we last sent (if we are
+	// channel initiator) or received (if non-initiator) in an update fee
+	// message, which haven't yet been included in a commitment.
+	// It will be nil if no fee update is un-committed.
+	pendingFeeUpdate *btcutil.Amount
+
+	// pendingAckFeeUpdate is set to the last committed fee update which is not
+	// yet ACKed. Set to nil if no such update.
+	pendingAckFeeUpdate *btcutil.Amount
```

Comment:
Are two variables required for this case? Seems that we could get by with only a single instance (the one above).